src/v3/src/components/LoopbackProbe/LoopbackProbe.test.tsx

```diff
@@ -37,7 +37,7 @@ jest.mock('../../contexts', () => ({
 jest.mock('../../../../util/Logger');
 
 describe('LoopbackProbe', () => {
-  const server: SetupServerApi = setupServer();
+  const server: SetupServer = setupServer();
```

Comment:
was this the wrong type before?

Reply:
There was a type error being thrown because `SetupServerApi` was expecting a property that wasn't on `setupServer()`